apps/alerts/lib/alert.ex

```diff
@@ -289,7 +289,7 @@ defimpl Poison.Encoder, for: Alerts.Alert do
   end
 
   defp format_time(t) do
-    case Timex.format(t, "{YYYY}-{M}-{D} {h24}:{m}") do
+    case Timex.format(t, "{YYYY}-{0M}-{0D}T{0h24}:{0m}:{0s}{Z:}") do
```

Comment:
what's the difference between 0m and the previous m?

Reply:
It is for the [formatting](https://hexdocs.pm/timex/Timex.Format.DateTime.Formatters.Default.html).  It pads the hours, minutes, and seconds with leading zeros.
So instead of `2023-5-4T1:3:4-04:00` it becomes `2021-05-04T01:03:04-04:00`